basics-assignment-3-start 3/src/app/app.component.html

```diff
@@ -0,0 +1,21 @@
+<!-- <div class="container">
+  <div class="row">
+    <div class="col-xs-12">
+      <ol>
+        <li>Add A button which says 'Display Details'</li>
+        <li>Add a paragraph with any content of your choice (e.g. 'Secret Password = tuna')</li>
+        <li>Toggle the displaying of that paragraph with the button created in the first step</li>
+        <li>Log all button clicks in an array and output that array below the secret paragraph (maybe log a timestamp or simply an incrementing number)</li>
+        <li>Starting at the 5th log item, give all future log items a blue background (via ngStyle) and white color (ngClass)</li>
+      </ol>
+    </div>
+  </div>
+</div> -->
+
+<button (click)="onClick()">Display Details</button>
+
+<p *ngIf="showContent">{{ paragraph }}</p>
+
+<div>
+  <p *ngFor="let counter of getRepeatArray(click)" [ngClass]="{ 'white-text': counter >= 5 }" [ngStyle]="{'background-color': counter >= 5 ? setBackgroundColor(): 'transparent'}">{{ counter }}</p>
```

Comment:
setBackgroundColor() no reason to use a fuction that always returns a constant string